backend/src/database/outcomes.go

```diff
@@ -18,6 +18,11 @@ func (db *DB) GetOutcomesForUser(id uint, page, perPage int, outcomeType models.
 	if err := query.Count(&count).Error; err != nil {
 		return 0, nil, err
 	}
+
+	if order != "" && orderBy != "" {
```

Comment:
if you look at the PR i have in right now #311 I created something like this 
```go
func validOrder(str string) string {
	if slices.Contains([]string{"asc", "desc"}, strings.ToLower(str)) {
		return strings.ToLower(str)
	}
	return "desc"
}
```
which you can use `query = query.Order(orderBy + " " + validOrder(order))

but also take a look to see how i sanitized the `order` string, with a map of all the possibilities. can we do something similar?

Reply:
Absolutely.  That fieldmap with an 'ok' is a nice pattern I like how that looks; glad you pointed it out.

Reply:
Should be all set.